cmd/quadlet/main.go

```diff
@@ -77,11 +77,10 @@ func Logf(format string, a ...interface{}) {
 	s := fmt.Sprintf(format, a...)
 	line := fmt.Sprintf("quadlet-generator[%d]: %s", os.Getpid(), s)
 
-	if !logToKmsg(line) {
-		// If we can't log, print to stderr
-		fmt.Fprintf(os.Stderr, "%s\n", line)
-		os.Stderr.Sync()
-	}
+	logToKmsg(line)
+
+	fmt.Fprintf(os.Stderr, "%s\n", line)
+	os.Stderr.Sync()
```

Comment:
@rhatdan, can you elaborate on changing the logic here?

Reply:
If you are running --dryrun you will not see the errors without this change. They will go directly to the log files.  It does not effect running it under daemon-reload.